### PR TITLE
Fix Dependabot config

### PR DESCRIPTION
**Pull request by Malax**
#170 had an invalid Dependabot config.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -13,7 +13,8 @@ updates:
         update-types:
           - "minor"
           - "patch"
-        patterns: "org.apache.maven.plugins:*"
+        patterns:
+	  - "org.apache.maven.plugins:*"
```
Comment (edmorley) [inline .github/dependabot.yml:17]: Will Dependabot mind this under-indentation? (I can't remember if it matters for yaml)

**Review by edmorley — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (Malax) [reply .github/dependabot.yml:17]: Good question, the validator check is not added for this PR (and the last one) :(. But the indentation is at least consistent throughout this file (2 spaces). A generic YAML validator says it's valid. 🤞🏻

Comment (edmorley) [reply .github/dependabot.yml:17]: > But the indentation is at least consistent throughout this file (2 spaces)

It's not consistent for the `- "org.apache.maven.plugins:*"` line for me?

Comment (Malax) [reply .github/dependabot.yml:17]: Yeah, and GitHub also doesn't like the file. I suspect some whitespace shenanigans is going on here with my editor.

**Pull request merged**